src/content/configuration/stats.md

```diff
@@ -13,6 +13,8 @@ The `stats` option lets you precisely control what bundle information gets displ
 
 T> For webpack-dev-server, this property needs to be in the `devServer` object.
 
+T> Some plugins, such as `webpack-dev-middleware`, intercept `stats`. Therefore you must set `stats` in that plugin's config.
```

Comment:
I'm a little confused about what this is suggesting. Would this be targeted at plugin authors who want to intercept `stats`? E.g. _why_ would I want to set `stats` in that plugin's config?

Reply:
I set `stats` to hide all warnings in my webpack config, but I was still getting warnings. I then tried to suppress all stats because I thought I was going crazy, but that didn't work either. Finally I discovered that `webpack-dev-middleware` was the plugin that was displaying the stats and overriding what I set in my webpack config. This was annoying to track down and not plainly obvious, especially for devs who are jumping into a complex webpack project